jetty-core/jetty-http/src/test/java/org/eclipse/jetty/http/HttpGeneratorClientTest.java

```diff
@@ -98,7 +98,7 @@ public void testEmptyHeaders() throws Exception
 
         HttpFields.Mutable fields = HttpFields.build();
         fields.add("Host", "something");
-        assertThrows(IllegalArgumentException.class, () -> fields.add("Null", null));
+        assertThrows(IllegalArgumentException.class, () -> fields.add("Null", (String)null));
```

Comment:
I fear this is a breaking change, as now existing code that passes `null` has to add a cast and recompile.

Reply:
Existing code that passes null to add will throw IAE, so I think this is OK.